smarts/core/sensors.py

```diff
@@ -909,6 +911,115 @@ def teardown(self):
         pass
 
 
+class UTurnTrajectorySensor(Sensor):
+    def __init__(self, vehicle, sim, mission_planner):
+        self._vehicle = vehicle
+        self._sim = sim
+        self._mission_planner = mission_planner
+        self._horizon = 50
+
+    def __call__(self):
+        # TODO: Reuse codes from RoadWaypointsSensor,
```

Comment:
@iman512003 I believe we need to tune the trajectory. The flaws I can see now: 1. the agent won't land in the middle of the target lane 2. the trajectories of incoming edge and oncoming edge are not consistent.